src/main/java/net/dv8tion/jda/internal/requests/Route.java

```diff
@@ -227,6 +227,17 @@
         public static final Route DELETE_INVITE =       new Route(DELETE, "invites/{code}");
     }
 
+    public static class Templates
+    {
+        public static final Route GET_TEMPLATE =               new Route(GET,    "guilds/templates/{code}");
```

Comment:
Imo it would make more sense to put these under the guild namespace since they all start with guild/

Reply:
i don't think it matters, as role endpoints are also not under the same namespace